pkg/plugin/util/exec_test.go

```diff
@@ -0,0 +1,48 @@
+package util
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestRunCmd(t *testing.T) {
```

Comment:
See that we use the following libs to do the tests. Could you please follow the same standard to do this one?

> 	. "github.com/onsi/ginkgo/v2"
> 	. "github.com/onsi/gomega"

Example:
https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/util/helpers_test.go

Reply:
Sure will do that